examples/counter/android/src/main/java/org/reduxkotlin/example/counter/MainActivity.kt

```diff
@@ -0,0 +1,49 @@
+package org.reduxkotlin.example.counter
+
+import android.os.Bundle
+import android.os.Handler
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_main.*
+import org.reduxkotlin.StoreSubscription
+import org.reduxkotlin.createStore
+import org.reduxkotlin.examples.counter.Decrement
+import org.reduxkotlin.examples.counter.Increment
+import org.reduxkotlin.examples.counter.reducer
+
+/**
+ * This is a sample of basic redux behavior.
+ * This is NOT best practice for structuring a multiplatform App.
```

Comment:
Is there a plan to show best practices?

Reply:
Yes, through other examples and documentation on the website.  This is strictly a port of the redux counter example.  I wanted to get something up as a template for other examples.  These will also let us test/exercise the core redux code within the repo.